website/data/docs-remote-plugins.json

```diff
@@ -71,6 +71,12 @@
     "repo": "hashicorp/packer-plugin-googlecompute",
     "version": "latest"
   },
+  {
+    "title": "HashiCups",
+    "path": "hashicups",
+    "repo": "hashicorp/packer-plugin-hashicups",
+    "version": "latest"
+  },
```

Comment:
I wonder if this should be a community plugin, given it's an example 🤔. Though we do maintain it !